lib/extensions/acts_as_helpers/active_record/base.rb

```diff
@@ -18,5 +18,14 @@ module ExperiencePointsInstanceMethods
     def manually_awarded?
       false
     end
+
+    # The description to be displayed in the 'reason' column for the experience points record on
+    # the experience points history page.
+    # This method is meant to be overridden by each class that acts as an experience points record.
+    #
+    # @return [String] Reason which will be displayed
+    def points_display_reason
+      raise NotImplementedError
```

Comment:
Give a reason why this is raised so people can grep the source code to figure this call site.